deploy/tests/schema/_posts.yml

```diff
@@ -0,0 +1,8 @@
+title: "String"
+config:
+  path: "guides"
+  ignore:
+    - "README.md"
+    - ..
+    - .
```

Comment:
Are these really necessary? Kind of wild if the plugin follows these paths.

Reply:
I can't remember exactly what the gem is doing but we certainly found out the hard way that these are required after trying to exclude them to begin with.